controllers/servers/utils.py

```diff
@@ -577,6 +577,16 @@ def validate_addons_request(request, replication_type):
     if request.volume_id == "" or (replication_type == REPLICATION_TYPE_MIRROR and request.replication_id == ""):
         raise ValidationException(messages.VOLUME_ID_SHOULD_NOT_BE_EMPTY_MESSAGE)
 
+    if replication_type == REPLICATION_TYPE_EAR:
+        logger.debug("validating obsolete rcrelationship parameters")
```

Comment:
IDK if "rcrelationship parameters" say anything to a client.. maybe non-EAR of something like that?

Reply:
Done